packages/cxl-ui/package.json

```diff
@@ -15,6 +15,7 @@
     "@pika/plugin-build-web": "^0.9.2",
     "@pika/plugin-bundle-web": "^0.9.2",
     "@pika/plugin-standard-pkg": "^0.9.2",
+    "@polymer/polymer": "^3.4.1",
```

Comment:
Why this explicit dependency added - I would think it's pulled in automagically by vaadin-button?

Reply:
`commit-lint` wasn't wasn't allowing me to commit without it.

Reply:
Can you post full output? I can't think of any rule that would block here.

Reply:
```
/mnt/c/Users/Andy/Web Development/repositories/anoblet/aybolit/packages/cxl-ui/src/components/cxl-link-button.js
  2:22  error  '@polymer/polymer' should be listed in the project's dependencies. Run 'npm i -S @polymer/polymer' to add it  import/no-extraneous-dependencies
```